RideShare/src/app/services/AllDriversServices/all-drivers.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { User } from '../../models/user.model';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AllDriversService {
+  /**
+   * Server and port number of the request
+   */
+  readonly port = 'http://localhost:3001';
```

Comment:
move to environment variable

Reply:
Changed to environment variable in environments.ts.